src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $week-day, $index in $start-day {
+    &--start-day-#{$week-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
The addition of `$index` to the `margin-left` calculation seems unnecessary unless `$index` includes a unit that represents a gap size. If `$index` is just a number without units, it should not be added to a size value. If it represents a gap, it should be clearly named, such as `$gapSize`.